### DISCUSSION TITLE
new performance measurements and comparative tests with other products

### DISCUSSION THREAD

**Discussion by alexted** [dragonflydb/dragonfly#996] [General]
Are you planning new performance measurements and comparative tests with other products? 
I would like to see fresh performance data on Dragonfly. 
In addition to the typical Redis, KeyDB and Memcached I would like to see a comparison with [Tarantool](https://github.com/tarantool/tarantool).
Should we expect it in the near future?

**Comment by romange** [comment]
Alex, Tarantool is an awesome project but we won't have the bandwidth to benchmark ourselves with every key-value store in the world. Take into account that tarantool is single threaded and it's not a pure in-memory store, therefore, I do not think there will be any surprises if you just comare `ops/sec` metric. On the other hand, Tarantool provides a rich set of capabilities that Dragonfly does not have. 

To conclude, these are two different products with different APIs and protocols, and comparing ops/sec head-to-head with Tarantool won't be that interesting.
